File: quiz 2/quizQ10.js
// Write a program to count the number of vowels in a given string.

// Will need to tell the computer which letters are vowels
// How to count the number of ocurences instead of just returning the vowels themselves
//may need to break the string down into an array of letters that are then compared to the vowels array

var string = "Angelica";

var vowels = /[AEIOUaeiou]/g;

console.log("There are " + string.match(vowels).length + " vowels in " + string);

File: quiz3/python/quizQ10.py
# Write a program to count the number of vowels in a given string.

# Will need to tell the computer which letters are vowels
# Increment the number of vowels with each one it finds

string = "krista"


def howMany():
    numVowels = 0
    for char in string:
        if char in "aeiouAEIOU":
            numVowels += 1
    print numVowels

howMany()